### ISSUE TITLE
Provider danbooru has stopped working with http return code 500

### ISSUE THREAD

**Issue by JMM72** [Nandaka/DanbooruDownloader#92]
Using 201061009. When trying to download from provider danbooru.donmai.us, error 500 (Internal server error) appears. The server was up and working while testing. Disabling compression, cookie, or downgrading to 20160816 doesn't solve the issue. Other providers (e.g. yande.re) work without problems.

**Comment by Nandaka** [comment]
Working on my side:
![image](https://cloud.githubusercontent.com/assets/836867/19494508/133769b2-95b2-11e6-9a38-57f3077b1ea4.png)

Check if you can access https://danbooru.donmai.us/posts.xml?tags=rating:s
also remember, free user can only search up to 2 keyword.

**Comment by JMM72** [comment]
Well that was embarassing, Murphy must be messing up. I was looking at the providers.xml and without doing any change to my setup, it all started working again. I don't know what could be happening, I copied the URLs being pulled from the log and I could access them on the browser but not from the program. After so many tries I decided it might be something about how danbooru was accepting the requests so I filed the bug, but yes it's solved now, sorry for the false alarm.
